09-EjercicioEnClase/Mariana Ocampo/index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/09-EjercicioEnClase/Mariana Ocampo/styles/style.css">
+    <title>Medicina eficaz</title>
+</head>
+
+<header>
+     Medicina eficaz
+</header>
+<body>
+
+    
+
+    <form class="Pagina">
+        <label for="Nombre">Nombre</label>
+        
+        <input name="ingresanombre" type="text" value="Por favor, ingresa aquí" id="Nombre"/>
```

Comment:
recuerda que el name que le pones al input debe ser el mismo que el del for en el label.